packages/cache/redis/src/index.ts

```diff
@@ -41,7 +40,6 @@ export default class RedisCache<V = string> implements KeyValueCache<V> {
           host: parsedHost,
           port: parseInt(parsedPort),
           password: parsedPassword,
-          lazyConnect: true,
```

Comment:
I think this doesn't make it optional. It completely disables it.

Reply:
Shoud I add a new parameter for this to the plugin itself to configure this from meshrc config file?

Reply:
I added a separate config for lazyConnect. And now it's set to true by default. So no breaking change. But it can be set to false in case someone else wants to use Azure or any other similar Redis providers.